homeassistant/config_entries.py

```diff
@@ -1874,6 +1874,12 @@ async def async_forward_entry_setup(
         component also has related platforms, the component will have to
         forward the entry to be setup by that component.
         """
+        return await self._async_forward_entry_setup(entry, domain, True)
+
+    async def _async_forward_entry_setup(
+        self, entry: ConfigEntry, domain: Platform | str, preload_platform: bool
+    ) -> bool:
```

Comment:
I didn't want to expose the `preload_platform` in the public api so I made `_async_forward_entry_setup`